README.md

```diff
@@ -47,6 +47,10 @@ $ sudo pacman -S gcc cmake sdl2 inotify-tools libxml2
 - [Visual Studio 2015+](https://visualstudio.microsoft.com/)
 - [SDL2 VC Development Libraries](https://www.libsdl.org/release/SDL2-devel-2.0.9-VC.zip)
 
+### MinGW
```

Comment:
MinGW should be a subsection of Windows. Also the Windows section should have a Visual Studio subsection for the current content.